### ISSUE TITLE
Can not 're-aquire' MO2 handle nexus links

### ISSUE THREAD

**Issue by LoneYukon6512** [TanninOne/modorganizer#898]
I use MO2 for FO4 and MO1 for Skyrim...
ran into an issue where, after modding up my Skyrim and switching back to MO2 to up date FO4 mods, I had to re-install MO2 with the option to handle Nexus links upon install before it would actually do so - kept getting the "No application set to handle operation' dialogue when trying to download from Nexus prior to re-install.

**Comment by Gernash** [comment]
run nxmhandler.exe in MO2 root folder

**Comment by LoneYukon6512** [comment]
kk...will do next time...tyvm

**Comment by GSDFan** [comment]
Instead of reinstalling MO try going to Settings, Nexus tab and press the Associate with "Download with manager" links button when you switch instances of MO.

**Comment by LoneYukon6512** [comment]
@GSDFan ....apparently you didn't read the whole post - I tried that.

**Comment by GSDFan** [comment]
Sorry i have been up since midnight, pulled a 12 hour shift and still have not had any sleep. 

As LoneYukon6512 said run nxmhandler.exe and see which game is missing. Make sure to add the correct version of MO for the game that you need.

**Comment by LoneYukon6512** [comment]
yeah...you better get some sleep dude...I am Loneyukon6512 - it was Gernash that told me to try that...thanks tho
